lib/WP_Auth0_Options.php

```diff
@@ -129,92 +134,124 @@ public function get_logout_url() {
 		return add_query_arg( 'action', 'logout', site_url( 'wp-login.php', 'login' ) );
 	}
 
+	/**
+	 * Get as a string-separated string and parse to array
+	 *
+	 * @return array
+	 */
+	public function get_lock_connections() {
+		$connections = $this->get( 'lock_connections' );
+		$connections = empty( $connections ) ? array() : explode( ',', $connections );
+		return array_map( 'trim', $connections );
+	}
+
+	/**
+	 * Add a new connection to the lock_connections setting
+	 *
+	 * @param string $add_conn - connection name to add
+	 */
+	public function add_lock_connection( $add_conn ) {
+		$connections = $this->get_lock_connections();
+
+		// Add if it doesn't exist already
+		if ( ! array_key_exists( $add_conn, $connections ) ) {
+			$connections[] = $add_conn;
+			$connections = implode( ',', $connections );
+			$this->set( 'lock_connections', $connections );
+		}
+	}
+
+	/**
+	 * Default settings when plugin is installed or reset
+	 *
+	 * @return array
+	 */
 	protected function defaults() {
 		return array(
+
+			// System
 			'version' => 1,
-			'metrics' => 1,
 			'last_step' => 1,
-			'auto_login' => 0,
-			'auto_login_method' => '',
+			'migration_token_id' => null,
+			'use_lock_10' => true,
+			'jwt_auth_integration' => false,
+			'amplificator_title' => '',
+			'amplificator_subtitle' => '',
+			'connections' => array(),
+			'auth0js-cdn' => '//cdn.auth0.com/js/auth0/9.1/auth0.min.js',
```

Comment:
you are using `https`, right?

Reply:
Some folks are, some are not ... though this URL will always be `https` so I guess that could be explicit here? This is just a reordering, FYI

Reply:
it's hosted by auth0 so I don't see a reason why they shouldn't be using `https`. They don't need to setup anything on their side. It's not like a custom domain where the certificate needs to be handled by your or something like that.. I'd enforce https on all auth0 urls. Double check with luis, but i think that's the right choice.

Reply:
this should be always https